src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -136,7 +136,7 @@ export async function waitForProjectReady(directory: string) {
 export async function waitForProjectsConfigured() {
     await pollUntil(() => {
         const files = glob.sync(`**/${projectConfigFile}`, { cwd: testAppsRoot});
-        return files.length === 16;
+        return files.length === 17;
```

Comment:
Can we have some sort of a Debug Assert message here so that we don't get a generic `Before all hook failed` the next time someone adds a test project?